packages/rrweb/src/record/mutation.ts

```diff
@@ -509,26 +509,33 @@ export default class MutationBuffer {
           ) {
             item.attributes.style = {};
           }
-          const styleObj = item.attributes.style as styleAttributeValue;
-          for (const pname of Array.from(target.style)) {
-            const newValue = target.style.getPropertyValue(pname);
-            const newPriority = target.style.getPropertyPriority(pname);
-            if (
-              newValue !== old.style.getPropertyValue(pname) ||
-              newPriority !== old.style.getPropertyPriority(pname)
-            ) {
-              if (newPriority === '') {
-                styleObj[pname] = newValue;
-              } else {
-                styleObj[pname] = [newValue, newPriority];
+          try {
+            const styleObj = item.attributes.style as styleAttributeValue;
+            for (const pname of Array.from(target.style)) {
+              const newValue = target.style.getPropertyValue(pname);
+              const newPriority = target.style.getPropertyPriority(pname);
+              if (
+                newValue !== old.style.getPropertyValue(pname) ||
+                newPriority !== old.style.getPropertyPriority(pname)
+              ) {
+                if (newPriority === '') {
+                  styleObj[pname] = newValue;
+                } else {
+                  styleObj[pname] = [newValue, newPriority];
+                }
               }
             }
-          }
-          for (const pname of Array.from(old.style)) {
-            if (target.style.getPropertyValue(pname) === '') {
-              // "if not set, returns the empty string"
-              styleObj[pname] = false; // delete
+            for (const pname of Array.from(old.style)) {
+              if (target.style.getPropertyValue(pname) === '') {
+                // "if not set, returns the empty string"
+                styleObj[pname] = false; // delete
+              }
             }
+          } catch (error) {
+            console.warn(
+              'Error when parsing update to style attribute:',
```

Comment:
Not sure if we do this elsewhere but should we add an rrweb-prefix? Just as a hint for users and for us where this is coming from 🤔 
```suggestion
              '[rrweb] Error when parsing update to style attribute:',
```

Reply:
Good point! We don't have this _yet_ in other places, but may have. It would be nice to tie in the sentry debugger here 😅 but I guess that'd be a bit trickier.